client/dashboard/plugins/schedule/index.tsx

```diff
@@ -1,11 +1,32 @@
+import { useNavigate } from '@tanstack/react-router';
+import { Button } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
+import { pluginsScheduleRoute } from '../../app/router/plugins';
 import { PageHeader } from '../../components/page-header';
 import PageLayout from '../../components/page-layout';
 
 export default function PluginsSchedule() {
+	const navigate = useNavigate( { from: pluginsScheduleRoute.fullPath } );
+
 	return (
-		<PageLayout size="small" header={ <PageHeader title={ __( 'Scheduled updates' ) } /> }>
-			{ /* TODO: Implement schedule updates UI */ }
+		<PageLayout
+			size="small"
+			header={
+				<PageHeader
+					title={ __( 'Scheduled updates' ) }
+					actions={
+						<Button
+							variant="primary"
+							onClick={ () => navigate( { to: '/plugins/schedule/new' } ) }
```

Comment:
Non-blocking

Instead of specifying the path we could use the defined route, that way if we ever change the path we don't need to update this component. Something like this:
```
onClick={ () navigate({ to: pluginsScheduleNewRoute.to }) 
```

Reply:
Absolutely. Good thinking. :-)